osu.Game/Overlays/BeatmapListing/BeatmapSearchGeneralFilterRow.cs

```diff
@@ -135,14 +141,28 @@ protected override void LoadComplete()
             {
                 base.LoadComplete();
 
+                config.BindWith(OsuSetting.BeatmapListingFeaturedArtistFilter, Active);
                 disclaimerShown = sessionStatics.GetBindable<bool>(Static.FeaturedArtistDisclaimerShownOnce);
+
+                // no need to show the disclaimer if the user already had it toggled off in config.
+                if (!Active.Value)
+                    disclaimerShown.Value = true;
+
+                if (game.LimitedToFeaturedArtists)
+                {
+                    Enabled.Value = false;
+                    Active.Disabled = true;
+                }
```

Comment:
Needs to have:

```cs
Active.Value = true;
```

Without it, anyone who has disabled this filter in config (eg. current android players) will be stuck with it disabled even if `LimitedToFeaturedArtists == true` (eg. android players when we release on google play).

Reply:
Android is not relevant for now, adjustments can be made via config migration or otherwise when that happens.